client/components/RelatedProductsAndOutfits/RelatedProducts.jsx

```diff
@@ -21,8 +21,8 @@ componentDidUpdate(prevProps) {
 }
 setProductsShown() {
   this.setState({
-    productsShown: this.props.relatedProducts.slice(0, 5),
-    hiddenRight: this.props.relatedProducts.slice(5)
+    productsShown: this.props.relatedProducts.slice(0, 4),
+    hiddenRight: this.props.relatedProducts.slice(4)
```

Comment:
might need do the scrolling with css, because the number of cards shown depends on the screen size?